src/main/java/racingcar/domain/Cars.java

```diff
@@ -1,20 +1,35 @@
 package racingcar.domain;
 
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
 
 public class Cars {
 
     private final List<Car> cars;
 
-    public Cars(int size, CarMoveCondition moveCondition) {
+    public Cars(String[] names) {
         this.cars = new ArrayList<>();
-        for (int i = 0; i < size; i++) {
-            this.cars.add(new Car(moveCondition));
+        for (String name : names) {
+            this.cars.add(new Car(new Name(name)));
         }
     }
 
+    // Constructor for UnitTest
+    public Cars(Car... cars) {
+        this.cars = Arrays.asList(cars);
```

Comment:
다른 생성자를 호출하는 방식으로 변경해보면 어떨까요?
```suggestion
        this(Arrays.asList(cars));
```